Sources/AppcuesKit/Data/Models/Experiment.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Experiment.swift
+//  AppcuesKit
+//
+//  Created by James Ellis on 10/17/22.
+//  Copyright © 2022 Appcues. All rights reserved.
+//
+
+import Foundation
+
+internal struct Experiment: Decodable {
+    let group: String
```

Comment:
fyi, updated this to just allow any String here, not requiring "control" or "exposed" enum types - my hope is to possibly add a little future-proofing for any multivariate experiments or different naming conventions in the future.  It still checks for `"control"` explicitly in the `ExperienceRenderer` to know if it should block the experience from rending, as a special case.